### ISSUE TITLE
Dark arrow in xfce4-panel Notification Area

### ISSUE THREAD

**Issue by malysps** [adapta-project/adapta-gtk-theme#289]
**Reporter info**
 ```
 * Distribution - Xubuntu 16.10 64-bit
 * Gtk+ 3.0 version - 3.20
 * Desktop environment - Xfce 4.12
 ```

**Adapta version**
3.89.1.5-0ubuntu1~yakkety1 (from PPA)

**Related Application and/or Shell theming (or name of widget)**
Xfce Panel, Notification Area

**Actual issue**
I have hidden icons in the Notification Area and the arrow is barely visible. On other themes it looks OK, so I assume it's not an icon theme problem and it could be related strictly to Adapta.

**Expected behavior**
Arrow should be white, light-gray or teal.

**Additional info**
Adapta-Nokto variant is OK and doesn't have this issue.

Great theme! I hope you support Xfce.
Screenshot attached.

![adapta-arrow](https://cloud.githubusercontent.com/assets/16068365/20517213/6b899d08-b09a-11e6-9561-7b107390dcf2.png)

**Comment by tista500** [comment]
@malysps,

Let me know this issue happens in both Adapta and Adapta-Nokto...
And hopefully you knew that arrow was rendered with Gtk+ 2 or 3 (GtkExpander's arrow?)...

Anyway I need to read the codes of xfce4-panel.

**Comment by tista500** [comment]
This one? `xfce4-panel/plugins/systray/systray.c`?
```
static void
systray_plugin_button_set_arrow (SystrayPlugin *plugin)
{
  GtkArrowType   arrow_type;
  gboolean       show_hidden;
  GtkOrientation orientation;

  panel_return_if_fail (XFCE_IS_SYSTRAY_PLUGIN (plugin));

  show_hidden = systray_box_get_show_hidden (XFCE_SYSTRAY_BOX (plugin->box));
  orientation = xfce_panel_plugin_get_orientation (XFCE_PANEL_PLUGIN (plugin));
  if (orientation == GTK_ORIENTATION_HORIZONTAL)
    arrow_type = show_hidden ? GTK_ARROW_LEFT : GTK_ARROW_RIGHT;
  else
    arrow_type = show_hidden ? GTK_ARROW_UP : GTK_ARROW_DOWN;

  xfce_arrow_button_set_arrow_type (XFCE_ARROW_BUTTON (plugin->button), arrow_t$
}
```

**Comment by malysps** [comment]
That arrow in Adapta-Nokto is OK, it's light-gray and clearly visible, as it should be.

I'm not sure about *systray.c*, but it looks it could be it.

**Comment by malysps** [comment]
That commit fixed this issue, thank you!

Sorry, I should be more specific about this hidden icons, you can easily choose which icon you want to hide in the settings window, it isn't necessary to have a lot of them - http://docs.xfce.org/xfce/xfce4-panel/systray

When do you plan to include this fix in the PPA for yakkety?

**Comment by tista500** [comment]
> it isn't necessary to have a lot of them - http://docs.xfce.org/xfce/xfce4-panel/systray

Ah, I now understood... 

> When do you plan to include this fix in the PPA for yakkety?

That commit is included 3.89.1.25 release: https://launchpad.net/~tista/+archive/ubuntu/adapta/+build/11239712

Regards.
